src/components/flag/__tests__/flag-test.js

```diff
@@ -1,9 +1,9 @@
 /** @jsx React.DOM */
 
-jest.dontMock('../code/views/FlagComponentView.jsx');
+jest.dontMock('../code/views/flagView.jsx');
 
 var React = require('react/addons');
-var FlagComponent = require('../code/views/FlagComponentView.jsx');
+var FlagComponent = require('../code/views/flagView.jsx');
```

Comment:
Do we have naming conventions laid out somewhere yet? @jodiedoubleday can you add a JIRA to create a CONTRIBUTING.mkd file for this repo?